### PR TITLE
Add daily notification for collection of ogre arrows from Rantz.

### LINKED ISSUE #8581: Daily Task Indicator - Ogre Arrows
Another useful feature to add to the daily task indicator options is picking up Ogre Arrows from Rantz if one has the Western Provinces diaries completed, since this is worth as much if not more than the other daily item pickups awarded by diaries!

### PR DESCRIPTION
**Pull request by Reasel**
Adds a notification for ogre arrow collection from Rantz with completion of at least easy western provinces diary.

Closes #8581

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java [hunk 1]
```diff
@@ -247,6 +253,21 @@ else if (dailyReset)
 		}
 	}
 
+	private void checkArrows(boolean dailyReset)
+	{
+		if (client.getVar(Varbits.DIARY_WESTERN_EASY) == 1)
+		{
+			if (client.getVar(Varbits.DAILY_ARROWS_STATE) == 0)
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java]: `if (dailyReset || client.getVar(Varbits.DAILY_ARROWS_STATE) == 0)`

Both checks have the same body, so there's no reason to have them in separate if branches.

Comment (Reasel) [reply runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java]: I simply matched the file format that was already there. Should I fix the other functions as well? Only in places where this is also true?

**Comment by Reasel** [comment]
Need to wait to check if the arrows can be claimed without a re-log.

File: runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksConfig.java [hunk 2]
```diff
@@ -109,4 +109,15 @@ default boolean showBonemeal()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+			position = 8,
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksConfig.java]: Fix your indetnation

**Comment by Reasel** [comment]
The collection is similar to the battle staff collection in that it requires a relog in order to claim. I have added the necessary logic to show the play this. I think this is good to go.

**Comment by xDemoN** [comment]
@Reasel Needs rebased

**Comment by deathbeam** [comment]
needs rebase

**Comment by deathbeam** [comment]
Also, @Adam- said he wants this off by default.

We already have a lot of daily notifs on by default, and this is not useful, so default off makes sense.

**Pull request closed without merging**